keyboards/prototypist/pt60/info.json

```
@@ -0,0 +1,97 @@
+{
+    "manufacturer": "Proto[Typist]",
+    "keyboard_name": "PT-60",
+    "maintainer": "Anjheos",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["B12", "B14", "A8", "A9", "A15", "B3", "B7", "B4", "B5", "B6", "A4", "A5", "A6", "A7" ],
+        "rows": ["B0", "B1", "B2", "B10", "B11"]
+    },
+    "processor": "STM32F303",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0001",
+        "vid": "0x4A46"
+    },
+    "layouts": {
+        "LAYOUT_all": {
```

Generally, it's just `LAYOUT` if there is only one layout, and `LAYOUT_all` when the are multiple layouts and this one would have all possible positions listed (even if they couldn't normally co-exist).

```suggestion
        "LAYOUT: {
```

This would also need all of the keymaps changed, too.

That said, if this supports the 60% iso/ansi layouts, those should be added (as well as community layout support enabled)